frontend/src/components/board/Card.js

```diff
@@ -67,7 +67,7 @@ TargetIndicator.propTypes = {
   parentLaneId: PropTypes.number,
 };
 
-class Card extends Component {
+class Card extends PureComponent {
```

Comment:
wdyt about converting this one to a functional component too?
looks like a easy win.

Reply:
Yeah, I went this way but it didn't turn out to be easy. In this case we have to use the hooks API which for some reason doesn't really want to work with our setup. I didn't want to spend any more time figuring why the drag and drop elements are treated as the same thing so reverted back to class component.